assets/scss/modules/mixins/_color.scss

```diff
@@ -0,0 +1,34 @@
+$colours:
```

Comment:
I would prefer using `map` instead of a list as a good practice.

Reply:
I made the change with the good practice you told me!

Reply:
GJ once again, @IgnacioRodrigues ! You are doing some cool Sass magic here, man! 👏